specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -49,4 +49,77 @@ scope http
             HttpHeader[] headers;
         }
     }
+
+    scope event
+    {
+        enum HttpEventType (uint8)
+        {
+            CORS_PREFLIGHT_REQUEST_NOT_ALLOWED (1),
+            CORS_REQUEST_NOT_ALLOWED (2),
+            GUARD_AUTHORIZATION_FAILURE (3),
+            GUARD_AUTHORIZATION_SUCCESS (4),
```

Comment:
Suggest we drop the `GUARD_` prefix here.

Reply:
OK